unit/src/TestISoilUtils.C

```diff
@@ -0,0 +1,244 @@
+// MOOSE includes
```

Comment:
These unit tests are great, thanks for taking the time to do this work. It certainly makes the code much better.